src/tools/common/dotnet-tool.ts

```diff
@@ -228,14 +226,51 @@ export abstract class DotnetTool implements IDotnetTool {
         return path.normalize(workDir)
     }
 
+    private async getQueryService(): Promise<string | null> {
+        // Use dotnet tool to get the first enabled nuget source.
+        const result = await this.execute(
+            'dotnet',
+            new ArgumentsBuilder().addArgument('nuget').addArgument('list').addArgument('source').addKeyValue('format', 'short').build()
```

Comment:
please move this to a variable instead, and if possible put each fluent call on a separate line (only if prettier or eslint does not complain)

Reply:
I had them on a separate line, but the linter moved it back on one line.